cmd/migrate/gear/core/1536736169_initialize_schema.up.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE _sky_user (
```

Comment:
What is the definition of core table (a.k.a. table starts with `_sky`)?

I guess it is not the responsibility of gears? (Because it would be hard not to understand `disabled` is the responsibility of auth instead of core lib), so maybe the definition is: **all gears' table shall not be accessed by other gears (even auth), except those tables under the `_sky` namespace.**?

Reply:
> all gears' table shall not be accessed by other gears (even auth), except those tables under the _sky namespace.

yes.

Maybe some confusion here. See if these updates would make things better:

- Move the migration files from `cmd/migrate/gear/core/` to `cmd/migrate/core`
- Update the command to `go run cmd/migrate/main.go -path cmd/migrate/core -schema app_helloworld -core up`